crates/bevy_tasks/src/task_pool.rs

```diff
@@ -265,35 +343,42 @@ impl Drop for TaskPool {
 ///
 /// For more information, see [`TaskPool::scope`].
 #[derive(Debug)]
-pub struct Scope<'scope, T> {
-    executor: &'scope async_executor::Executor<'scope>,
-    local_executor: &'scope async_executor::LocalExecutor<'scope>,
-    spawned: Vec<async_executor::Task<T>>,
+pub struct Scope<'scope, 'env: 'scope, T> {
```

Comment:
Usability point: now that we support nesting. Should this be Clone?

Reply:
The user only get an `&Scope`, and they should just pass that around when using it.  Feels like cloning Scope would just be slower, since you'd need to clone all 3 of the underlying references.